src/intTest/resources/outbound_uat_data/acceptance/type1-birth-all-fields.edifact.dat

```diff
@@ -0,0 +1,24 @@
+UNB+UNOA:2+TES5+XX11+200610:1438+00000001'
+UNH+00000001+FHSREG:0:1:FH:FHS001'
+BGM+++507'
+NAD+FHS+XX1:954'
+DTM+137:202006101438:203'
+RFF+950:G1'
+S01+1'
+RFF+TN:1'
+NAD+GP+4826940,281:900'
+NAD+RIC+RT:956'
+HEA+ACD+A:ZZZ'
+HEA+ATP+1:ZZZ'
+HEA+DM+Y:ZZZ'
+DTM+956:19920113:102'
+LOC+950+BURY'
+FTX+RGI+++FREE TEXT VALUE TEST ABCD'
+S02+2'
+PNA+PAT+N/10/10:OPI+++SU:STEVENS+FO:CHARLES+TI:MR+MI:ANTHONY+FS:JOHN'
+PNA+PER+N/10/10:OPI+++SU:STEVENS_OLD'
```

Comment:
underscore isn't a valid character here. Just make up some previous name instead?

-	Have alphabetic, SPACE, "'" and "-" characters only.
-	Have a maximum length of 35 characters (Previous Surnames of more than 35 characters MUST be truncated for these purposes).
-	Have alphabetic characters in upper case.